vyper/parser/stmt.py

```diff
@@ -203,41 +203,45 @@ def ann_assign(self):
             varname = self.stmt.target.id
             pos = self.context.new_variable(varname, typ)
             o = LLLnode.from_list('pass', typ=None, pos=pos)
-            if self.stmt.value is not None:
-                sub = Expr(self.stmt.value, self.context).lll_node
+            if self.stmt.value is None:
+                raise StructureException(
+                    'Fresh variables must be initialized explicitly',
```

Comment:
![fresh](https://user-images.githubusercontent.com/3859395/65531239-75ef8380-dec7-11e9-9789-d2dd403752bf.jpeg)
```suggestion
                    'New variables must be initialized explicitly',
```